app/src/__snapshots__/routes.test.tsx.snap

```diff
@@ -0,0 +1,165 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`routes Test Enabled Routes 1`] = `
```

Comment:
@ashfahan how can we reducer this snapshot

Reply:
we cant reduce this snapshot more or less it is the default routes generate for default roles

Reply:
would an equality assertion work here, i.e. instead of using a snapshot

Reply:
@ashfahan this had been marked as resolved, I do not think its resolved.

Reply:
we can use equality assertion. one way or the other the result would remain same .. so should i ?

Reply:
please use an equality assertion for such cases.